phoenix-core/src/it/java/org/apache/phoenix/end2end/index/MutableIndexSplitIT.java

```diff
@@ -135,9 +135,11 @@ private void createTableAndLoadData(Connection conn1, String tableName, String i
         List<RegionInfo> regionsOfUserTable = null;
         for(int i = 0; i <=1; i++) {
             boolean split = false;
-            for (int j = 0; j < 60 && !split; j++) {
+            for (int j = 0; j < 150 && !split; j++) {
                 try {
                     if (localIndex) {
+                        //With Hbase 2.2 the local index splits trigger longCompactions, and have to wait for an
```

Comment:
Can we change the period of this discharger? That would prevent this wait+retry to be copied to the N+1th test case.

Reply:
I'd prefer to keep the changes minimal for this patch.
But I'll make a separate ticket for that.

Reply:
https://issues.apache.org/jira/browse/PHOENIX-5699